From oliver@neukum.org Wed Jul  4 08:11:52 2007
From: Andrey Arapov <andrey.arapov@gmail.com>
From: Oliver Neukum <oliver@neukum.org>
Date: Wed, 4 Jul 2007 17:11:42 +0200
Subject: USB: cdc-acm: add new device id to option driver
To: gregkh@suse.de, linux-usb-devel@lists.sourceforge.net
Message-ID: <200707041711.42532.oliver@neukum.org>

From: Andrey Arapov <andrey.arapov@gmail.com>

USB: add new device id to option driver
device is Samsung X180 China cellphone

Signed-off-by: Andrey Arapov <andrey.arapov@gmail.com>
Acked-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/class/cdc-acm.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/usb/class/cdc-acm.c
+++ b/drivers/usb/class/cdc-acm.c
@@ -1157,6 +1157,9 @@ static struct usb_device_id acm_ids[] = 
 	{ USB_DEVICE(0x0870, 0x0001), /* Metricom GS Modem */
 	.driver_info = NO_UNION_NORMAL, /* has no union descriptor */
 	},
+	{ USB_DEVICE(0x0e8d, 0x0003), /* FIREFLY, MediaTek Inc; andrey.arapov@gmail.com */
+	.driver_info = NO_UNION_NORMAL, /* has no union descriptor */
+	},
 	{ USB_DEVICE(0x0482, 0x0203), /* KYOCERA AH-K3001V */
 	.driver_info = NO_UNION_NORMAL, /* has no union descriptor */
 	},
